.github/workflows/ci.yml

```diff
@@ -110,12 +110,12 @@ jobs:
           - crate: picky-asn1-x509
             msvc: "1.60"
           - crate: picky
-            msvc: "1.60"
+            msvc: "1.65"
```

Comment:
Out of curiosity, what caused this to be bumped? I assume the new dependencies?
README should be updated as well:
https://github.com/Devolutions/picky-rs/blob/308c09ae786d0166f1bcbe6abcba576d324426b0/picky/README.md?plain=1#L5

Reply:
Updated `p256`/`p384` crates transitive dependencies require >= 1.65